common/common_test.go

```diff
@@ -383,125 +369,37 @@ func getblockStub(method string, params []json.RawMessage) (json.RawMessage, err
 	if method != "getblock" {
 		testT.Error("unexpected method")
 	}
-	var height string
-	err := json.Unmarshal(params[0], &height)
+	var arg string
+	err := json.Unmarshal(params[0], &arg)
 	if err != nil {
 		testT.Fatal("could not unmarshal height")
 	}
 
 	step++
 	switch step {
 	case 1:
-		if height != "380640" {
-			testT.Error("unexpected height")
+		return []byte("{\"Tx\": [\"00\"], \"Hash\": \"0000380640\"}"), nil
+	case 2:
+		if arg != "0000380640" {
+			testT.Error("unexpected hash")
 		}
 		// Sunny-day
 		return blocks[0], nil
-	case 2:
-		return []byte("{\"Tx\": [\"00\"]}"), nil
 	case 3:
-		if height != "380641" {
-			testT.Error("unexpected height")
+		return []byte("{\"Tx\": [\"00\"], \"Hash\": \"0000380641\"}"), nil
+	case 4:
+		if arg != "0000380641" {
+			testT.Error("unexpected hash")
 		}
 		// Sunny-day
 		return blocks[1], nil
-	case 4:
-		return []byte("{\"Tx\": [\"00\"]}"), nil
 	case 5:
-		if height != "380642" {
-			testT.Error("unexpected height", height)
+		if arg != "380642" {
+			testT.Error("unexpected hash")
 		}
-		// Simulate that we're synced (caught up);
+		// Simulate that we're synced (caught up, latest block 380641);
 		// this should cause one 10s sleep (then retry).
```

Comment:
```suggestion
		// this should cause one sleep (then retry).
```
The sleep interval isn't 10s.

Reply:
Good catch, and actually there isn't any sleep-retry in this case (I single-stepped in the debugger to make sure), so I removed that comment line. I think that comment was copy-pasted from elsewhere.